.github/workflows/deploy-to-s3.yml

```diff
@@ -0,0 +1,41 @@
+name: Deploy to S3
+# Workflow copied from https://github.com/cowprotocol/token-lists/blob/main/.github/workflows/s3Deploy.yml
+
+on:
+  release:
+    types: [created]
+  # Allow manual trigger
+  workflow_dispatch:
+
+# Required for authenticating with AWS IAM
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+        with:
+          persist-credentials: false
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@7474bc4690e29a8392af63c5b98e7449536d5c3a # v4.3.1
+        with:
+          role-to-assume: arn:aws:iam::693696263829:role/token-list-github-action-role
+          role-session-name: githubactionsession
+          aws-region: eu-central-1
+
+      - name: Upload to S3
+        run: |
+          aws s3 sync src s3://files.cow.fi/cow-sdk --delete
```

Comment:
question:  I don't have a context for this task. Why we upload the src folder there instead of dist(built of the sdk)?